distribution/carbon-home/bin/dropins.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
```

Comment:
This file should be included in the Kernel feature and not in the distribution of the source.